src/actions/audit-log-actions.js

```diff
@@ -88,7 +88,8 @@ export const getAuditLog =
       page,
       per_page: perPage,
       expand: "user",
-      access_token: accessToken
+      access_token: accessToken,
+      summitTZ
```

Comment:
we dont do this .... we use UTC epoch please do check other services
the conversion to summit time zone should be done at front end